core/templates/services/contextual/url.service.ts

```diff
@@ -34,10 +34,18 @@ interface UrlParamsType {
 export class UrlService {
   constructor(private windowRef: WindowRef) {}
 
+  /**
+   * Returns the current location.
```

Comment:
Try to write docstrings as "nouns", i.e. don't use verbs or things that imply verbs. Here and below. The returns part is already clear -- you can rephrase this as: This function is used to find the current location of a window.